docs/Homebrew-Governance.md

```diff
@@ -48,13 +46,11 @@
 
 4. Any approved amendments will take effect three weeks after the close of voting.
 
-5. No amendment may be proposed which violates Software Freedom Conservancy policies. The PLC may veto such an amendment by an ordinary resolution.
-
 ## 4. Project Leadership Committee
 
-1. The governance and operation of Homebrew is determined by the PLC. The PLC will represent Homebrew in all dealings with the Software Freedom Conservancy (SFC). The PLC has final authority in all matters related to the operation of Homebrew, and will comply with all Software Freedom Conservancy policies.
+1. The financial administration of Homebrew, organisation of the AGM, enforcement of the code of conduct and removal of members are performed by the PLC. The PLC will represent Homebrew in all dealings with OpenCollective.
 
-2. The PLC consists of five members. Committee members are elected by Homebrew members in a [Meek Single Transferable Vote](https://en.wikipedia.org/wiki/Counting_single_transferable_votes#Meek) election using the Droop quota. Each PLC member will serve a term of two years or until the member's successor is elected. Any sudden vacancy in the PLC will be filled by the usual procedure for electing PLC members at the next general meeting, typically the next AGM.
+2. The PLC consists of five members including the Project Leader. Committee members are elected by majority vote of Homebrew members. Each PLC member will serve a term of two years or until the member's successor is elected. Any sudden vacancy in the PLC will be filled by the usual procedure for electing PLC members at the next general meeting, typically the next AGM.
```

Comment:
Sorry, one more question. Are we switching the election type again back to a majority vote kind of situation, or is this just removing clutter from the document? Same for the PL below.

Reply:
Note sure I understand the question, can you elaborate further?

Reply:
`Meek Single Transferable Vote](https://en.wikipedia.org/wiki/Counting_single_transferable_votes#Meek) election using the Droop quota` is not strictly the same as` majority vote of Homebrew members`

Reply:
It looks like the election types were changed in a PR to this repo: https://github.com/Homebrew/brew/pull/10637. So this document was out of sync with the private document. So now, when the private document was copied here, those changes were lost.

Reply:
It might be too late but given the number of nominations is equal to the number of seats, the result of a majority vote and a STV should be the same? So what I am saying is that we could count the votes this time under majority rules and fix this for next time.

Reply:
Yeah, it doesn't make a difference this time around. I'll try to remember to open a PR aligning the two that we can vote on next time

Reply:
> It looks like the election types were changed in a PR to _this_ repo: #10637 (instead of going first through the private one). So this document was out of sync with the private document. So now, when the private document was copied here, those changes were lost.

Ooops. We should address that after this is merged.

Reply:
Done by @issyl0 in https://github.com/Homebrew/brew/pull/13118